zeal.js

```diff
@@ -342,6 +342,19 @@ $("#clearterm").on("click", function() {
     terminal.reset();
 });
 
+var recterm = false;
+var reced_array = [];
+
+$("#recterm").on("click", function() {
+    recterm =! recterm;
```

Comment:
Can you create a function that would call the UART component?
```
$("#recterm").on("click", function() {
    zealcom.uart.toggleRecord("uart_received.bin");
    // If "recording" class was set, it will be removed, else it'll be added
    $(this).toggleClass("recording");
```

Of course, you'll need to modify a bit the CSS to have `.recording` class instead that would change the color of the button (red?)